src/components/Dashboard/TodoList.vue

```diff
@@ -133,91 +154,115 @@
                     @click.native.stop="doResumePayment(task)"
-                    data-test-id="btn-resume-payment"
                   >
-                    <span>Resume Payment</span>
-                    <!-- Cancel Payment -->
+                    <span>Retry Payment</span>
                   </v-btn>
-                  <v-menu offset-y left>
-                    <template v-slot:activator="{ on }">
-                      <v-btn color="primary"
-                        v-on="on" id="pending-item-menu-activator" :disabled="!task.enabled"
-                        class="actions__more-actions__btn px-0"
-                        @click.native.stop
-                        data-test-id="btn-pending-filing-menu"
-                      >
-                        <v-icon>mdi-menu-down</v-icon>
-                      </v-btn>
-                    </template>
-                    <v-list ref="pending_actions" class="actions__more-actions">
-                      <v-list-item id="btn-cancel-payment" @click="confirmCancelPayment(task)"
-                        data-test-id="btn-cancel-payment"
-                      >
-                        <v-list-item-title>Cancel Payment</v-list-item-title>
-                      </v-list-item>
-                    </v-list>
-                  </v-menu>
-                </template>
 
-                <v-btn v-else-if="isError(task)"
-                  class="btn-retry-payment"
-                  color="primary"
-                  :disabled="!task.enabled"
-                  @click.native.stop="doResumePayment(task)"
-                >
-                  <span>Retry Payment</span>
-                </v-btn>
+                  <template v-else-if="isPaid(task)">
+                    <!-- no action button in this case -->
+                  </template>
 
-                <template v-else-if="isPaid(task)">
-                  <!-- no action button in this case -->
-                </template>
-
-                <v-btn v-else-if="!isCompleted(task)"
+                  <v-btn v-else-if="!isCompleted(task)"
                   class="btn-file-now"
                   color="primary"
                   :disabled="!task.enabled || coaPending || !confirmCheckbox || hasBlockerFiling"
                   @click.native.stop="doFileNow(task)"
                 >
                   <span>File Annual Report</span>
                 </v-btn>
+                </div>
               </div>
             </div>
           </div>
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
-
-          <div v-if="isCorrection(task)" data-test-class="correction-comment" class="todo-list-detail">
+          <div v-if="isCorrection(task)">
+            <div v-if="isDraft(task)" data-test-class="correction-draft" class="todo-list-detail">
+              <p class="list-item__subtitle">
+                This filing is incomplete. The payment appears to have been unsuccessful for some reason.
```

Comment:
This sentence may or may not be true...
- if the staff saved a draft to resume later, this statement is false
- if our API returned with a filing error and saved a draft, this statement is true

You probably need to discuss this with Scott, unless you feel pre-emptive and want to make the second sentence conditional for example.

Reply:
K, will discuss with Scott and confirm design, thanks!

Reply:
I see the Draft text is updated in a future commit... so this conversation is resolved!